app/models/licence_transaction.rb

```diff
@@ -21,9 +22,17 @@ def self.title
     "Licence"
   end
 
+  def schema_organisations
```

Comment:
I think we probably shouldn't be using `schema_organisations` as the organisation is not being pulled by the schema in this case so is a little confusing. Maybe just 'organisations'? We'd also need to update the [parent method](https://github.com/alphagov/specialist-publisher/blob/7f67d74dba73813b9740e628806fc775f435de9c/app/models/document.rb#L189) and the references to it within the codebase